<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactItem</class>
 <widget class="QWidget" name="ContactItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item alignment="Qt::AlignmentFlag::AlignVCenter">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <widget class="QLabel" name="icon_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>头像</string>
      </property>
     </widget>
     <widget class="QLabel" name="dot_lab">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>-4</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/res/red_point.png</pixmap>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="name_lab">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
